plenum/test/view_change_slow_nodes/test_view_change_gc_all_nodes_random_delay.py

```diff
@@ -113,7 +110,6 @@ def checkPrePrepareSentAtLeastByPrimary():
     checkViewNoForNodes(txnPoolNodeSet, viewNo)
     last_ordered_3pc = (viewNo, batches_count)
     check_nodes_last_ordered_3pc(txnPoolNodeSet, last_ordered_3pc)
-    check_nodes_requests_size(txnPoolNodeSet, 1)
```

Comment:
Why did we remove this checks?

Reply:
For now we change requests size only after checkpoint stabilization not after VC